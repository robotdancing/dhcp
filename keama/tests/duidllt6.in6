# LLT server duid config

# authoritative is mandatory
authoritative;

# LLT server duid declaration
server-duid llt;

