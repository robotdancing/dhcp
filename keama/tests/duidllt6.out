{
  # LLT server duid config
  # authoritative is mandatory
  "Dhcp6": {
    # LLT server duid declaration
    "server-id": {
      "type": "LLT"
    }
  }
}
